### ISSUE TITLE
NoMethodError on 429 Response

### ISSUE THREAD

**Issue by subakva** [DripEmail/drip-ruby#24]
When the rate limit is exceeded, the drip API server responds with this content:

```
{"message":"API rate limit exceeded. Please try again in an hour."}
```

When the client library tries to create a resource from the response, it causes a NoMethodError:

```
NoMethodError: undefined method `keys' for "API rate limit exceeded. Please try again in an hour.":String
File "/app/vendor/bundle/ruby/2.3.0/gems/drip-ruby-0.0.10/lib/drip/resource.rb" line 32 in block in process
File "/app/vendor/bundle/ruby/2.3.0/gems/drip-ruby-0.0.10/lib/drip/resource.rb" line 31 in tap
File "/app/vendor/bundle/ruby/2.3.0/gems/drip-ruby-0.0.10/lib/drip/resource.rb" line 31 in process
File "/app/vendor/bundle/ruby/2.3.0/gems/drip-ruby-0.0.10/lib/drip/resource.rb" line 9 in initialize
File "/app/vendor/bundle/ruby/2.3.0/gems/drip-ruby-0.0.10/lib/drip/response.rb" line 56 in new
File "/app/vendor/bundle/ruby/2.3.0/gems/drip-ruby-0.0.10/lib/drip/response.rb" line 56 in block (2 levels) in parse_members
File "/app/vendor/bundle/ruby/2.3.0/gems/drip-ruby-0.0.10/lib/drip/response.rb" line 49 in each
File "/app/vendor/bundle/ruby/2.3.0/gems/drip-ruby-0.0.10/lib/drip/response.rb" line 49 in block in parse_members
File "/app/vendor/bundle/ruby/2.3.0/gems/drip-ruby-0.0.10/lib/drip/response.rb" line 47 in tap
File "/app/vendor/bundle/ruby/2.3.0/gems/drip-ruby-0.0.10/lib/drip/response.rb" line 47 in parse_members
File "/app/vendor/bundle/ruby/2.3.0/gems/drip-ruby-0.0.10/lib/drip/response.rb" line 13 in initialize
File "/app/vendor/bundle/ruby/2.3.0/gems/drip-ruby-0.0.10/lib/drip/client.rb" line 70 in new
File "/app/vendor/bundle/ruby/2.3.0/gems/drip-ruby-0.0.10/lib/drip/client.rb" line 70 in build_response
File "/app/vendor/bundle/ruby/2.3.0/gems/drip-ruby-0.0.10/lib/drip/client.rb" line 55 in make_request
File "/app/vendor/bundle/ruby/2.3.0/gems/drip-ruby-0.0.10/lib/drip/client.rb" line 43 in post
File "/app/vendor/bundle/ruby/2.3.0/gems/drip-ruby-0.0.10/lib/drip/client/subscribers.rb" line 69 in create_or_update_subscribers
```

**Comment by djreimer** [comment]
Thanks for reporting! Will look into a fix.

**Comment by subdigital** [comment]
I'm also hitting this.  I want my jobs to be smart and not keep retrying. If this were a specific error, I could rescue it and queue the job to happen in 1 hour instead.

**Comment by will-in-wi** [comment]
It looks like this got fixed in 33c6de65eb42733b6eb0898f5262d1f701e9a7fe. Is this still an issue?
